### ISSUE TITLE
ManagerInterface

### ISSUE THREAD

**Issue by revilon1991** [silasjoisten/sonata-multiupload-bundle#48]
@sonata.media.manager.media with doctrine_orm have default instance Sonata\MediaBundle\Entity\MediaManager which implement Sonata\Doctrine\Model\ManagerInterface

Replace statement from
src/Controller/MultiUploadController.php:6

`use Sonata\CoreBundle\Model\ManagerInterface;`
to
`use Sonata\Doctrine\Model\ManagerInterface;`

**Comment by OskarStark** [comment]
That’s correct, but you also need to require the latest core bundle in composer.json and create a new release.
A Major, because it is a BC break
